button-react.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```

Comment:
I think this file should be in `src` folder

Reply:
right